hack/update-deps.sh

```diff
@@ -51,22 +51,9 @@ shift 4
 cd "$BUILD_WORKSPACE_DIRECTORY"
 trap 'echo "FAILED" >&2' ERR
 
-prune-vendor() {
-  find vendor -type f \
-    -not -iname "*.c" \
-    -not -iname "*.go" \
-    -not -iname "*.h" \
-    -not -iname "*.proto" \
-    -not -iname "*.s" \
-    -not -iname "AUTHORS*" \
-    -not -iname "CONTRIBUTORS*" \
-    -not -iname "COPYING*" \
-    -not -iname "LICENSE*" \
-    -not -iname "NOTICE*" \
-    -delete
-}
-
 export GO111MODULE=on
+export GOPROXY=https://proxy.golang.org
```

Comment:
`go get -u` works differently with this setting, so we want to do it before the `case` below